### PR TITLE
Limit the Full-Width Responsive upgrade in Auto Ads to mobile users o…

### PR DESCRIPTION
**Pull request by Jiaming-X**
…nly.

### CODE REVIEW

File: extensions/amp-auto-ads/0.1/placement.js [hunk 1]
```diff
@@ -199,13 +199,17 @@ export class Placement {
           this.state_ = PlacementState.TOO_NEAR_EXISTING_AD;
```
Comment (aishwaryavora) [inline extensions/amp-auto-ads/0.1/placement.js:199]: Can you please update the unit test for these changes? The test file is here: extensions/amp-auto-ads/0.1/test/test-placement.js.

Comment (Jiaming-X) [reply extensions/amp-auto-ads/0.1/placement.js:199]: Added.

**Comment by Jiaming-X** [comment]
Accidentally added other master commits here. Please ignore it.

**Comment by Jiaming-X** [comment]
Use https://github.com/ampproject/amphtml/pull/28376 instead

**Pull request closed without merging**